terraform/baremetal-incus/versions.tf

```diff
@@ -1,7 +1,9 @@
 terraform {
+  required_version = "1.6.2"
   required_providers {
     incus = {
-      source = "lxc/incus"
+      source  = "lxc/incus"
+      version = "0.1.0"
```

Comment:
Not sure about that since I'm deploying with a mix of Terraform and OpenTofu so the version requirement is going to become annoying.

Same thing on the Incus side. I think once we start needing a specific feature to have this work, we'll do a ">=" condition but until then, any version should work fine.